nautobot_golden_config/template_content.py

```diff
@@ -34,15 +35,18 @@ def right_page(self):
     def detail_tabs(self):
         """Add a Configuration Compliance tab to the Device detail view if the Configuration Compliance associated to it."""
         try:
-            return [
-                {
-                    "title": "Configuration Compliance",
-                    "url": reverse(
-                        "plugins:nautobot_golden_config:configcompliance_devicetab",
-                        kwargs={"pk": self.get_device().pk},
-                    ),
-                }
-            ]
+            if ConfigCompliance.objects.filter(device=self.device):
+                return [
+                    {
+                        "title": "Configuration Compliance",
+                        "url": reverse(
+                            "plugins:nautobot_golden_config:configcompliance_devicetab",
+                            kwargs={"pk": self.device.pk},
+                        ),
+                    }
+                ]
+            else:
+                return []
```

Comment:
```suggestion
            return []
```

Reply:
That should solve the failed pylint test.

Reply:
Moved this return outside of try/except.

Do You know why we have `ObjectDoesNotExist` exception, while we use `filter()` ? CC @itdependsnetworks

Reply:
you are adding it, it wasn't there before.

Reply:
> you are adding it, it wasn't there before.

https://github.com/nautobot/nautobot-app-golden-config/blame/develop/nautobot_golden_config/template_content.py#L46

Reply:
can you like the filter?